app/uk/gov/hmrc/plasticpackagingtax/registration/config/AppConfig.scala

```diff
@@ -175,4 +175,8 @@ class AppConfig @Inject() (config: Configuration, val servicesConfig: ServicesCo
   lazy val mfaUpliftUrl: String = config.get[String]("urls.mfaUplift")
 
   lazy val businessAccountUrl: String = config.get[String]("urls.businessAccount")
+
+  lazy val accessibilityStatementServicePath: String =
+    config.get[String]("accessibility-statement.service-path")
```

Comment:
What does dot service-path mean?

Reply:
It's to do with linking up with the cross-HMRC accessibility statement service - https://github.com/hmrc/accessibility-statement-frontend#users-of-play-ui-version-8120-or-above-or-play-frontend-hmrc-version-0190-or-above

Reply:
The url passed to GRS has to be {{/accessibility-statement/plastic-packaging-tax}}

Reply:
I'm going to pass `/plastic-packaging-tax`.
GRS do not appear to have implemented this yet; accessibilityUrl is abit of an ambiguous name (do they really want is to compose a full tax platform URL?).

 I believe it's intention is to replace the value the receiving application would have otherwise got from it's own `accessibility-statement.service-path` config. Hence passing the value of our `accessibility-statement.service-path` feels sensible.

Reply:
I checked with GRS yesterday and the link passed does have to be as per the ticket and my comment above.